src/app/directives/image/image.directive.ts

```diff
@@ -12,6 +12,12 @@ import { ImageSizes, ImageUrl } from "@interfaces/apiInterfaces";
 import { assetRoot } from "@services/config/config.service";
 import { OrderedSet } from "immutable";
 
+export const notFoundImage: ImageUrl = {
+  url: `${assetRoot}/images/404.png`,
+  height: 221,
```

Comment:
why 221?

Reply:
That is the height of the image in pixels

Reply:
two issues:

- the 404 image may not always be these dimensions
- why is it not 220? Why is it not square?